plasma_framework/contracts/src/exits/payment/routers/PaymentInFlightExitRouter.sol

```diff
@@ -193,9 +186,8 @@ contract PaymentInFlightExitRouter is
             erc20Vault: erc20Vault,
             safeGasStipend: paymentExitGameArgs.safeGasStipend
         });
-        startIFEBond = BondSize.buildParams(INITIAL_IFE_BOND_SIZE, BOND_LOWER_BOUND_DIVISOR, BOND_UPPER_BOUND_MULTIPLIER);
-        piggybackBond = BondSize.buildParams(INITIAL_PB_BOND_SIZE, BOND_LOWER_BOUND_DIVISOR, BOND_UPPER_BOUND_MULTIPLIER);
-        processIFEBounty = ExitBounty.buildParams(INITIAL_IFE_EXIT_BOUNTY_SIZE, EXIT_BOUNTY_LOWER_BOUND_DIVISOR, EXIT_BOUNTY_UPPER_BOUND_MULTIPLIER);
+        startIFEBond = BondSize.buildParams(INITIAL_IFE_BOND_SIZE, 0, BOND_LOWER_BOUND_DIVISOR, BOND_UPPER_BOUND_MULTIPLIER);
```

Comment:
hmm, I am thinking to rename some of the constants (probably need to port back to the spec too.)

I know we only have bounty for piggyback. However, might still be easier to read the code if there is 

`START_IFE_BOUNTY = 0` and `PIGGYBACK_BOUNTY = real_ife_bounty`. current var `INITIAL_IFE_EXIT_BOUNTY_SIZE` would let me link to start IFE instead of piggyback intuitively.

And with this, we can replace the magic number 0 to that constant for better readability.

Reply:
done `76e2c7b`